java/java/src/main/java/org/signal/internal/Native.java

```diff
@@ -97,12 +97,12 @@ private Native() {}
   public static native byte[] HKDF_DeriveSecrets(int version, byte[] inputKeyMaterial, byte[] salt, byte[] info, int outputLength);
 
   public static native long[] IdentityKeyPair_Deserialize(byte[] data);
-  public static native byte[] IdentityKeyPair_Serialize(long publicKeyHandle, long privateKeyHandle);
+  public static native byte[] IdentityKeyPair_Serialize(long privateKey, long publicKey);
 
   public static native void NumericFingerprintGenerator_Destroy(long handle);
   public static native String NumericFingerprintGenerator_GetDisplayString(long handle);
   public static native byte[] NumericFingerprintGenerator_GetScannableEncoding(long handle);
-  public static native long NumericFingerprintGenerator_New(int iterations, int version, byte[] localIdentifier, byte[] localKey, byte[] remoteIdentifier, byte[] remoteKey);
```

Comment:
Whoops, these do not have the same behavior in Java and C at the moment. I'll take it out on Tue.